runtime/gc_glue_java/CompactSchemeFixupObject.cpp

```diff
@@ -73,7 +73,7 @@ MM_CompactSchemeFixupObject::fixupContinuationObject(MM_EnvironmentStandard *env
 	fixupMixedObject(objectPtr);
 	/* fixup Java Stacks in J9VMContinuation */
 	J9VMThread *currentThread = (J9VMThread *)env->getLanguageVMThread();
-	if (VM_VMHelpers::needScanStacksForContinuation(currentThread, objectPtr)) {
+	if (VM_VMHelpers::needScanStacksForContinuation(currentThread, objectPtr, true)) {
```

Comment:
We need a comment here, something like: 
In sliding compaction we must fix slots exactly once. Since we will fixup stack slots of mounted Virtual threads later during root fixup, we will skip it during this heap fixup pass (hence passing true(false) for scanOnly(Dis)Mounted parameter)